cicd/build.sh

```diff
@@ -195,8 +195,8 @@ if [ "$DEV" == true ]; then
 else
     pip3 install --upgrade pip
     # add cortx-py-utils below
-    yum install -y cortx-py-utils 
-    yum install -y python36-cortx-prvsnr
+    yum install -y cortx-py-utils --nogpgcheck
+    yum install -y python36-cortx-prvsnr --nogpgcheck
```

Comment:
@udayan-y21  Please check and confirm if we still require cortx-prvsnr code in csm deployment.
Theoratically it should not be reuquired